src/mmrelay/plugin_loader.py

```
@@ -486,46 +763,60 @@ def clone_or_update_repo(repo_url, ref, plugins_dir):
 
-            if requirements:
-                if in_pipx:
-                    # Use pipx inject for each requirement
-                    logger.info(
-                        f"Installing requirements for plugin {repo_name} with pipx inject"
+            if in_pipx:
+                # Use pipx to install the requirements.txt
+                logger.info(f"Installing requirements for plugin {repo_name} with pipx")
+                pipx_path = shutil.which("pipx")
+                if not pipx_path:
+                    raise FileNotFoundError("pipx executable not found on PATH")
```

Ruff (0.13.1)

Abstract `raise` to an inner function

(TRY301)